### PR TITLE
Visits history

### PR DESCRIPTION
**Pull request by Johngtka**
Features List:
- [x] `added new history component`
- [x] `added empty state component`
- [x] `component can hide or show autocomplete when patient was selected in patient table by History button`
- [x] `when user click in history point in side menu shows autocomplete when patient was searched autocomplete hidden and show table`
- [x] `table shows patient visit history by patient id`
- [x] `in table is date with date pipe set on ( DD.MM.YYYY )`
- [x] `to empty state added inputs for next usage`
- [x] `when patient do not have visits open empty state component with translation`
- [x] `added styles to empty state component `

ezgif com-video-to-gif

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "MENU": {
         "HOME": "Home",
         "ABOUT": "About",
-        "VISIT": "Visit"
+        "VISIT": "Visit",
+        "HISTORY": "History"
```
Comment (macrog) [inline src/assets/i18n/en.json:9]: maybe this should be called 'Patient's history' or 'Previous appointments' ....

Comment (Johngtka) [reply src/assets/i18n/en.json:9]: or VISIT_HISTORY

**Comment by macrog** [comment]
change the icon please, use the same one we have in patient table 
image

**Comment by macrog** [comment]
When showing empty state it means that the selected patient doesn't have any previous appointments in our system. You should be able to see the autocomplete with a patient name, and be able to change a patient.

**Comment by macrog** [comment]
missing closing button 
image

File: src/app/empty.state/empty.state.component.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+.text {
+    font-size: 3vw;
+}
+mat-icon {
+    font-size: 7vw;
+    height: 7vw;
+    width: 7vw;
+    color: gray;
+}
```
Comment (macrog) [inline src/app/empty.state/empty.state.component.css:17]: prettier code

File: src/app/patient-table/patient-table.component.html [hunk 3]
```diff
@@ -58,14 +58,16 @@
             <button mat-icon-button [matMenuTriggerFor]="menu">
                 <mat-icon>more_vert</mat-icon>
             </button>
+            <!-- [disabled]="true" -->
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.html:61]: remove commented code

**Review by macrog — CHANGES_REQUESTED** [review]
just a couple of details, in general not bad

**Comment by Johngtka** [comment]
> change the icon please, use the same one we have in patient table image

it is the same icon used in patient table

**Comment by Johngtka** [comment]
> missing closing button image
> 
> Add as well on the top name and surname of the patient for which we are showing the table

done

**Comment by macrog** [comment]
> > change the icon please, use the same one we have in patient table image
> 
> it is the same icon used in patient table

in the patient table that is used to navigate to this page 😄 
image

**Comment by Johngtka** [comment]
> > > change the icon please, use the same one we have in patient table image
> > 
> > 
> > it is the same icon used in patient table
> 
> in the patient table that is used to navigate to this page 😄 image

done

**Comment by Johngtka** [comment]
S&M commit proposal:
`feat: new component for showing visits history (#42)`

**Review by macrog — APPROVED** [review]
works fine!

**Pull request merged**